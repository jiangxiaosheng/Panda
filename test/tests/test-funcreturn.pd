func foo(a: int): int {
    return a
}

func main() {
    foo(10)
}