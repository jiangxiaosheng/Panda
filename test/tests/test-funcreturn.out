Sementically checked program: 


int foo(a) {
return (int : a);
}

void main() {
(int : foo((int : 10)))
}

